src/parse-stream.js

```diff
@@ -325,6 +325,19 @@ export default class ParseStream extends Stream {
       });
       return;
     }
+    match = (/^#EXT-X-PROGRAM-DATE-TIME:?(.*)$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'date-time'
+      };
+      if (match[1]) {
+        event.dateTimeString = match[1];
+        event.dateTimeObject = new Date(match[1]);
```

Comment:
@nickygerritsen We only have the timezone of the user's current locale, not the source media's locale. As I mentioned before, the date-time is converted to whatever the user's locale is and the source locale would be lost. That's why I kept the string in there.

As for validation, I decided to just take whatever date-time string occurs if matched; it will just result in the `Date` object returning `Invalid Date` (or `NaN` if other functions are used). I thought the user should know if their `#EXT-X-PROGRAM-DATE-TIME` tag contained an invalid format instead of it being swept under the rug. They have the string for reference so they can see where the format went wrong and either manipulate it, correct it or ignore it.